dallinger/recruiters.py

```diff
@@ -80,6 +80,7 @@ class Recruiter(object):
 
     nickname = None
     external_submission_url = None  # MTurkRecruiter, for one, overides this
+    publish_experiment_default = True
```

Comment:
I suppose what we could even do here is set `publish_experiment_default = not supports_delayed_publishing`, then you don't need to override `publish_experiment_default` in the subclasses?

Reply:
I agree with this one :+1: